demo/MANIFEST.in

```diff
@@ -1,3 +0,0 @@
-recursive-include demoproject/templates *
```

Comment:
Why would you remove that?

Reply:
Not needed. Maybe I misunderstood what the contents of that folder were for, but it didn't work to run it using the included the instructions, and it does run now.